ui/src/datasets.tsx

```diff
@@ -163,204 +171,304 @@ export function Datasets() {
+                startIcon={<AddIcon />}
+                variant="contained"
+                onClick={() => {
+                  navigate(absoluteNewConceptSetURL(params));
+                }}
+              >
+                New data feature
+              </Button>
+            </GridLayout>
+            <Paper
+              sx={{
+                p: 2,
+                display: "block",
+                width: "100%",
+                height: "100%",
+              }}
+            >
+              <GridLayout cols="1fr 1fr" spacing={2}>
+                <GridLayout rows spacing={2}>
+                  <GridLayout cols>
+                    <Step
+                      index={0}
+                      active={conceptSetOccurrences.length === 0}
+                      completed={conceptSetOccurrences.length > 0}
+                    >
+                      <StepLabel>Select data features</StepLabel>
+                    </Step>
+                    <StepConnector />
+                  </GridLayout>
+                  <Paper
+                    variant="outlined"
+                    sx={{
+                      py: 2,
+                      display: "block",
+                      width: "100%",
+                      height: "100%",
+                    }}
+                  >
+                    <GridBox sx={{ px: 1, overflowY: "auto" }}>
+                      {workspaceConceptSets.data?.length === 0 ? (
+                        <GridLayout rows="1fr 70% 1fr">
+                          {underlay.uiConfiguration.prepackagedConceptSets ? (
+                            <GridLayout rows height="auto">
+                              {listConceptSets(
+                                false,
+                                underlay.uiConfiguration.prepackagedConceptSets
+                              )}
+                            </GridLayout>
+                          ) : (
+                            <GridBox />
+                          )}
+                          <GridBox
+                            sx={{
+                              boxShadow: (theme) =>
+                                `0 -1px 0 ${theme.palette.divider}`,
+                            }}
+                          >
+                            <Empty
+                              maxWidth="80%"
+                              title="Data features are categories of data to export"
+                              subtitle={
+                                <>
+                                  <Link
+                                    variant="link"
+                                    underline="hover"
+                                    component={RouterLink}
+                                    to={absoluteNewConceptSetURL(params)}
+                                  >
+                                    Create a new data feature
+                                  </Link>{" "}
+                                  to define more criteria
```

Comment:
I don't quite follow this text -- create a new concept set to define more criteria?

Reply:
Hmm no not really. Is this from the UX recommendations? If so, happy to defer to them.

Reply:
Is it just this one or the cohort text too? As I was implementing this I realized that data features were never empty because of "Demographics" and so were never described anywhere. I'll see what Yikai's thoughts are as well.

Reply:
Yeah both are a little confusing to me. Dropping the "to define [more] criteria" from both would make more sense to me -- or just a big Create new button or something. The descriptions of cohorts and data features are clear, and I think sufficient.